exchange/exchange.go

```diff
@@ -949,19 +957,29 @@ func (e *exchange) makeBid(bids []*pbsOrtbBid, auc *auction, returnCreative bool
 	return result, errs
 }
 
-func makeBidExtJSON(ext json.RawMessage, prebid *openrtb_ext.ExtBidPrebid) (json.RawMessage, error) {
-	// no existing bid.ext. generate a bid.ext with just our prebid section populated.
-	if len(ext) == 0 {
-		bidExt := &openrtb_ext.ExtBid{Prebid: prebid}
-		return json.Marshal(bidExt)
-	}
+func makeBidExtJSON(ext json.RawMessage, prebid *openrtb_ext.ExtBidPrebid, impExtInfo ImpExtInfo) (json.RawMessage, error) {
 
-	// update existing bid.ext with our prebid section. if bid.ext.prebid already exists, it will be overwritten.
 	var extMap map[string]interface{}
-	if err := json.Unmarshal(ext, &extMap); err != nil {
-		return nil, err
+
+	if len(ext) != 0 {
+		// update existing bid.ext with our prebid section. if bid.ext.prebid already exists, it will be overwritten.
```

Comment:
This comment is still valid. 
`if bid.ext.prebid already exists, it will be overwritten` - this is what it does. 
EchoVideoAttrs will be inserted in `bid.ext` not `bid.ext.prebid` in further processing. 
I'm open to modify this comment to anything better, please post your suggestions.

Reply:
Sorry, I should have been more clearer. I agree with what you said above. What I meant by my comment was that maybe we should extend that comment to also add a note about the video attributes. Something like:
```
// update existing bid.ext with our prebid section and add stored video 
// attributes for imp if echoVideoAttrs set to true.
// if bid.ext.prebid already exists, it will be overwritten.
```
Also, perhaps this comment makes more sense to be on the first line of this function